pkg/usage/usage.go

```diff
@@ -19,6 +19,7 @@ package usage
 import (
 	//node "github.com/openebs/jiva-operator/pkg/kubernetes/node"
 	k8sapi "github.com/openebs/lib-csi/pkg/client/k8s"
+	"github.com/schemahero/schemahero/pkg/logger"
```

Comment:
logrus `"github.com/sirupsen/logrus"` already been used in repo ,that can be used, also err check has been left intentionally to not log the usage error in first place, so i don't think this change required as well.